src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [movies] = useState(moviesFromServer);
```

Comment:
why would need a state without a state setter function?
i don't think you need one here.
Just `.filter` the `moviesFromServer`
```suggestion
```